### PR TITLE
Added a detailed description of the function's parameters and return …

### PR DESCRIPTION
**Pull request by Sjaure**
…value.

### 🎫 Ticket

[TICKET_ID]

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s).
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: changelog/fix-docblock-tribe_get_listview_args [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: tweak
+
+Add information and missing tags to �+� `tribe_get_listview_args()` docblock.
```
Comment (dpanta94) [inline changelog/fix-docblock-tribe_get_listview_args:4]: `Â+Â` what are these chars ? Maybe they are there by mistake ?

Comment (dpanta94) [inline changelog/fix-docblock-tribe_get_listview_args:4]:
```suggestion
Adds information and missing tags to �+� `tribe_get_listview_args()` docblock.
```

File: src/functions/template-tags/link.php [hunk 2]
```diff
@@ -250,15 +250,22 @@ function tribe_get_listview_dir_link( $direction = 'next', $term = null, $curren
...
+ * @param int         $page                 The current page number. Default is 1.
+ * @param string      $direction            The navigation direction, either 'next' or 'prev'. Default is 'next'.
+ * @param string|null $currently_displaying The view currently being displayed, such as 'list' or 'past'.
+ *                                          Default is null, which triggers a fallback to the default list view.
  *
- * @return array
+ * @return array { An associative array containing the updated display and page number.
+ *      @type string $display The view to be displayed, either 'list' or 'past'.
+ *      @type int    $page    The updated page number.
+ * }
```
Comment (dpanta94) [inline src/functions/template-tags/link.php:268]: I don't like this style BUT i am not sure what our docs say as well. Could you confirm if that's how its expected we style associative arrays in docblocks please ?

**Review by dpanta94 — CHANGES_REQUESTED** [review]

File: changelog/fix-docblock-tribe_get_listview_args (hunk 1 above)
Comment (Sjaure) [reply changelog/fix-docblock-tribe_get_listview_args:4]: These got in there when I copy/pasted the chengelog line. I will make sure I check next time.

File: src/functions/template-tags/link.php (hunk 2 above)
Comment (Sjaure) [reply src/functions/template-tags/link.php:268]: I'm adjusting based on this documentation: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#other-tags

**Review by dpanta94 — APPROVED** [review]

**Pull request merged**